cell/src/main/java/jetbrains/jetpad/cell/text/TextEditing.java

```diff
@@ -47,47 +47,11 @@
 
   private static final TextEditorStateHandler STATE_HANDLER = new TextEditorStateHandler(false, null);
 
-
-  public static boolean isTextEditor(Cell cell) {
-    return cell instanceof TextCell;
-  }
-
-  public static TextEditor textEditor(Cell cell) {
```

Comment:
I see dependencies on TextEditing.textEditor() in jetbrains.datapad.main.lang.editor.mapper.input.EmptyStatementCell and jetbrains.datapad.main.lang.editor.mapper.input.EmptyDrawOptionCell. They don't compile after this change. Looks like you need a datapad pull request.